src/internal/packager/helm/repo.go

```diff
@@ -143,6 +143,14 @@ func (h *Helm) DownloadPublishedChart(cosignKeyPath string) error {
 		chartURL  string
 		err       error
 	)
+	repoFile, err := repo.LoadFile(pull.Settings.RepositoryConfig)
+
+	if err != nil {
+		message.Debug("Unable to load the repo file at %q: %s", pull.Settings.RepositoryConfig, err.Error())
+	}
+
```

Comment:
I would leave a comment here denoting why this is not a failure case for this function, despite an `error` occurring, or return this error

Reply:
this also needs to be `DebugF`, not `Debug` if using `%q` and `%s`